UI/src/app/file-upload/file-upload.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, OnInit, OnDestroy, ViewChild, Input, Output } from '@angular/core';
+import { HttpClient, HttpHeaders, HttpEventType, HttpRequest, HttpErrorResponse, HttpEvent } from '@angular/common/http';
+import { EventEmitter } from 'events';
+import { FormControl, FormGroup, Validators, NgForm } from '@angular/forms';
+import { BackendConnectorService } from "../services";
+import { Dataset} from '../../../models';
+
+@Component({
+  selector: 'app-file-upload',
+  templateUrl: './file-upload.component.html',
+  styleUrls: ['./file-upload.component.css']
+})
+export class FileUploadComponent implements OnInit, OnDestroy {
+    fileSelected: boolean = true;
+    statusFormGroup: FormGroup;
+    fileToUpload: File  = null;
+    uploadProgress:number = 0;
+    uploadComplete:boolean = false;
+    uploadingProgressing:boolean = false;
+    fileUploadSub: any;
+    serverResponse: any;
+
+    @ViewChild('myInput') myFileInput: any;
```

Comment:
we should remove the unused imports and varaiables in here

Reply:
You are right. We have to remove unused variables